### DISCUSSION TITLE
error html/template: pattern matches no files: `/photoprism/assets/templates/*`

### DISCUSSION THREAD

**Discussion by Macchiato17** [photoprism/photoprism#2349] [Q&A] [answered]
Hi all,

unfortunately I have a problem with my photoprism installation, that I can't resolve on my own. Perhaps there is someone who may point me to some resource, give me a hint or whatever. I already checked the troubleshooting docs, but from my understanding my configuration should be fine. There are not "file-system full, access denied" or whatever messages in the syslog that would point me to any hint...

OK, the last "green" running image was **20220121**

Starting with **220302** the following error in the log showed up and prevented the docker image from starting

> photoprism
> init as root
> image: amd64-prod, build 220302-buster
> home: /photoprism
> umask: "0002" (u=rwx,g=rwx,o=rx)
> install-path: /opt/photoprism
> running as uid 0
> /opt/photoprism/bin/photoprism start
> **/entrypoint.sh: line 104: audit.sh: command not found**

After that version, I tried every new release / preview, since in some discussion I found an information that this issue will be solved in an upcoming preview release. Unfortunately those new (preview) builds continuously failed. The bullseye- and now bookworm-based previews failed to start showing always the same error message about some kind of missing files:

> started preview as root (amd64-prod)
> Problems? Our Troubleshooting Checklists help you quickly diagnose and solve them:
> https://docs.photoprism.app/getting-started/troubleshooting/
> file umask....: "0002" (u=rwx,g=rwx,o=rx)
> home directory: /photoprism
> assets path...: /photoprism/assets
> storage path..: /photoprism/storage
> config path...: default
> cache path....: default
> backup path...: /var/lib/photoprism
> import path...: /photoprism/import
> originals path: /photoprism/originals
> running as uid 0
> /opt/photoprism/bin/photoprism start
> time="2022-05-22T10:43:11Z" level=info msg="PhotoPrism® needs your support!"
> time="2022-05-22T10:43:11Z" level=info msg="Visit https://docs.photoprism.app/funding/ to learn more."
> time="2022-05-22T10:43:13Z" level=info msg="config: read-only mode enabled"
> **time="2022-05-22T10:43:13Z" level=error msg="html/template: pattern matches no files: `/photoprism/assets/templates/*`"**

I did some research but have not found any information how to solve this issue. So I would be really happy if someone might help me to overcome this obstacle. Thanks in advance 😃

**Comment by lastzero** [comment, marked as answer]
Using **Portainer**? It's a known issue that it remembers the environment variables of previously used images and applies them also to new images from the same repository:

- I'm not sure why that is, maybe to prevent data loss when users have specified config values for an old image they want to keep in the new version... however, these variables are internal and should not be set externally, except by experts in test & dev environments.
- For example, the path to the assets has changed from `/photoprism/assets` to `/opt/photoprism/assets`, so (accidentally) mounting `/photoprism` instead of `/photoprism/originals` doesn't break things anymore.
- If anyone knows how to prevent this behaviour, please let us know! 👍

**Reply by Macchiato17** [reply]
Thanks @lastzero for your quick reply, indeed I'm using **portainer** and your hint already was the solution: 

> path to the assets has changed from /photoprism/assets to /opt/photoprism/assets

It would be nice if an additional hint could be included in the error-message, allowing the user to get a clearer understanding what happened that made photoprism getting upset 😉 

Thanks so much again, now I'm part of this nice piece of software again. Have a good time - cheers
